packages/backend-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -107,6 +107,9 @@ describe('proxy', () => {
         test('onProxyReq', () => {
             const mockSetHeader = jest.fn();
 
+            onProxyReq({ setHeader: mockSetHeader as unknown } as ClientRequest);
```

Comment:
Can you explain this change, please?

Reply:
Oh wait, sorry, I get it. Minor suggestion, move the type case to line 108, so you don't need to do it in 110 and 113

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/714/commits/69bcf476dcf0e0569f56a9082d0193dd36225e67